### DISCUSSION TITLE
Support for Ubuntu 24.04

### DISCUSSION THREAD

**Discussion by Dalvany** [SeaQL/FireDBG.for.Rust#47] [Ideas]
Hello, I upgraded my Ubuntu to 24.04 and there is no pre-build binaries so VSCode keeps asking me to download the latest version. I can make a pull request if it's fine.

**Comment by billy1624** [comment]
Hey @Dalvany, yes please! A PR would be thankful!!

**Comment by Dalvany** [comment]
I made a PR : https://github.com/SeaQL/FireDBG.for.Rust/pull/48

**Reply by billy1624** [reply]
Thanks!! @Dalvany
